### ISSUE TITLE
DS-010: X1 Wrong pin name for Pin 1

### ISSUE THREAD

**Issue by mskarayigit** [pixhawk/Pixhawk-Standards#12]
<img width="1068" alt="Screen Shot 2020-12-09 at 09 46 44" src="https://user-images.githubusercontent.com/51109715/101595116-ef5b8f00-3a03-11eb-81f2-8e54596be0db.png">

I think this one should be FMU_CH8. Same error in the "FMUV6X_stm32_pinout RC01" table.

**Comment by jinger26** [comment]
@davids5 can you please review this?

**Comment by DuganY** [comment]
@mskarayigit you are correct, this should be FMU_CH8 in [DS-010.](https://docs.google.com/document/d/1uA9Afcg06xGI9gUycfzb4YRRTf_GvRx0ARZjqc_et8Y/edit?ts=60043e41#heading=h.fj4efant2uu6). Document has been updated.

**Comment by jinger26** [comment]
Thanks @DuganY . @LorenzMeier: need to make a revision and new release.
